R/uniDimBayesianOutputTables.R

```diff
@@ -249,3 +250,80 @@
 
   return()
 }
+
+
+.BayesianFitMeasuresTable <- function(jaspResults, model, options) {
+
+  if (!is.null(.getStateContainerB(jaspResults)[["fitTable"]]$object))
+    return()
+
+  fitTable <- createJaspTable(gettextf("Fit Measures for the Single-Factor Model"))
+
+  fitTable$dependOn(options = c("omegaScale", "fitMeasures", "fitCutoffSat", "fitCutoffNull", "pointEst"))
+
+  cred <- format(100 * options[["credibleIntervalValueItem"]], digits = 3, drop0trailing = TRUE)
+
+  fitTable$addColumnInfo(name = "measure", title = gettext("Fit measure"), type = "string")
+  fitTable$addColumnInfo(name = "pointEst", title = gettext("Point estimate"), type = "number")
+  fitTable$addColumnInfo(name = "cutoff",
+                         title = gettextf("Relative to cutoff"), type = "number")
```

Comment:
```suggestion
                         title = gettext("Relative to cutoff"), type = "number")
```
or did you mean to interpolate something?